server/src/main/java/org/apache/uniffle/server/netty/ShuffleServerNettyHandler.java

```diff
@@ -274,7 +273,7 @@ public void handleSendShuffleDataRequest(TransportClient client, SendShuffleData
           return;
         }
         final long start = System.currentTimeMillis();
-        shuffleBufferManager.releaseMemory(req.encodedLength(), false, true);
+        shuffleBufferManager.releaseMemory(req.getDecodedLength(), false, true);
```

Comment:
Worried about `req.getDecodedLength()` is not equals to `encodedLength` since the `encodedLength` is calculated approximately, but `req.getDecodedLength()` is calculated accurately.

So It is necessary to have a test to ensure the releaseMemory is equals to requireMemory, so that there is not memory leak take place.

Reply:
There already has a test class ```NettyProtocolTest``` to make sure that the encodedLength sent from client is same as  the getDecodedLength of the server received.
Does we need a new test class?

Reply:
@lwllvyb It is enough, thanks for show me this